validator/node/node.go

```diff
@@ -821,15 +818,13 @@ func (c *ValidatorClient) registerRPCGatewayService(router *mux.Router) error {
 		),
 		gwruntime.WithForwardResponseOption(gateway.HttpResponseModifier),
 	)
-	muxHandler := func(apiMware *apimiddleware.ApiProxyMiddleware, h http.HandlerFunc, w http.ResponseWriter, req *http.Request) {
+
+	// TODO: remove api middleware from this
+	muxHandler := func(_ *apimiddleware.ApiProxyMiddleware, h http.HandlerFunc, w http.ResponseWriter, req *http.Request) {
 		// The validator gateway handler requires this special logic as it serves two kinds of APIs, namely
-		// the standard validator keymanager API under the /eth namespace, and the Prysm internal
 		// validator API under the /api namespace. Finally, it also serves requests to host the validator web UI.
```

Comment:
Are these comments still correct?

Reply:
the validator API is still under /api. i guess during the transition period there are 3 kinds of APIs